src/analytics/SRGAnalytics.js

```diff
@@ -0,0 +1,873 @@
+import * as PlayerEvents from '../utils/PlayerEvents.js';
+import Pillarbox from '../pillarbox.js';
+/* eslint max-lines-per-function: ["error", 200] */
+/* eslint max-statements: ["error", 20]*/
+/* eslint complexity: ["error", 10]*/
+/**
+ * SRG analytics
+ * @ignore
+ *
+ * ### Script URL
+ * JS script : https://colibri-js.akamaized.net/penguin/tc_SRGGD_11.js
+ *
+ * ### Official documentation
+ * Variables list
+ * @see https://confluence.srg.beecollaboration.com/display/INTFORSCHUNG/Datalayer+for+media+players
+ *
+ * Standard event sequences
+ * @see https://confluence.srg.beecollaboration.com/display/INTFORSCHUNG/standard+streaming+events%3A+sequence+of+events+for+media+player+actions
+ *
+ * Review of Standard Media Actions
+ * @see https://confluence.srg.beecollaboration.com/display/INTFORSCHUNG/Implementation+Concept+-+draft
+ *
+ * ComScore Implementation Guide
+ * @see https://www.dropbox.com/sh/cdwuikq0abxi21m/AABmSyXYKUTWSAwRZgQA9Ujna/JavaScript%20Latest%20Version?dl=0&preview=Comscore_Library-JavaScript-Streaming_Tag-Implementation_Guide-International.pdf&subfolder_nav_tracking=1
+ *
+ * ### Variables list
+ * - 'event_id', // init | play | stop | pos | pause | seek | uptime | eof
+ * - 'event_timestamp', // Seems to be generated automaticly from the documentation, but the TP overrides it
+ * - 'event_name', // NA TP seems to not sending this variable
+ * - 'event_source', // NA TP seems to not sending this variable
+ * - 'event_name', // NA TP seems to not sending this variable
+ * - 'event_value', // NA TP seems to not sending this variable
+ * - 'navigation_environment', // prod | preprod
+ * - 'media_subtitles_on', // string true | false
+ * - 'media_timeshift', // need better description
+ * - 'media_quality', // SD | HD ?
+ * - 'media_bandwidth', // NA for the web, 64000
+ * - 'media_volume', // from 0 to 100
+ * - 'media_embedding_url', //
+ * - 'media_player_name', // videojs | letterbox-web ?
+ * - 'media_chromecast_selected', // boolean true | false
+ * - 'media_player_version', // player's version
+ * - 'media_player_display', // is the player mode, on the TP : inline, embed etc..
+ * - 'media_audio_track', // NA
+ * - 'media_position_real', // NA
+ * - 'media_time_spent', // NA
+ * - 'device_id', // NA
+ * - 'user_id_log_in', // NA only RTS has log in today
+ * - 'media_thumbnail', // Not required by the spec but sended by the TP
+ * - 'media_bu_distributer', // Not required by the spec but sended by the TP
+ *
+ *
+ * ### Sequence stories
+ *
+ * __Story 1 (AoD/VOD-basics)__: A VoD is played. The user does not interact with the player. The VoD plays to its end.
+ *
+ * Hints:
+ * - Media sessions allways start with PLAY. They end with STOP or EOF (or with PAUSE or last POS)
+ * - POS is sent ever 30s
```

Comment:
Not sure that providing use cases in the documentation is useful to the end user.